.github/workflows/delete-omnistrate-service-plan.yaml

```
@@ -47,6 +47,7 @@ jobs:
           - tier-name: free-${{ github.event_name == 'pull_request' && github.event.pull_request.head.ref || github.ref_name }}
           - tier-name: pro-${{ github.event_name == 'pull_request' && github.event.pull_request.head.ref || github.ref_name }}
           - tier-name: enterprise-${{ github.event_name == 'pull_request' && github.event.pull_request.head.ref || github.ref_name }}
+          - tier-name: startup-${{ github.event_name == 'pull_request' && github.event.pull_request.head.ref || github.ref_name }}
     steps:
       - name: Checkout
         uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)